libraries/hatch-server-user-management/src/UserServiceClient.ts

```diff
@@ -0,0 +1,20 @@
+export class UserInfo {
```

Comment:
Any reason these classes can't be immutable, i.e. use the constructor with readonly public params like BasicRouteParams, etc.?

Reply:
Updated